tox.ini

```diff
@@ -38,41 +36,32 @@ sitepackages = false
 deps =
     flake8>=3.6.0,<4
     yamllint>=1.11.1,<2
-extras =
 skip_install = true
 usedevelop = false
-
-[testenv:doc]
-# doc requires py3 due to use of f'' strings and using only python3 as
-# basepython risks using python3.4 which is not supported.
-basepython = python3.7
-passenv = *
 commands =
-    python setup.py build_sphinx -n -W --builder=html
-extras =
-    docs
+    flake8 piedpiper-noop-function
```

Comment:
This needs to either glob "piedpiper-noop" or we need to specify "piedpiper-noop-gateway" as well

Reply:
For sake of easy debugging in the future, a second `flake8` command was given in the `[testenv:lint]` checks.